UnityProject/Assets/Scripts/Clothing/FacehuggerImpregnation.cs

```diff
@@ -27,6 +27,9 @@ public class FacehuggerImpregnation : NetworkBehaviour, IServerInventoryMove, IC
 		[Tooltip("Reference to larvae gameObject so we can spawn it")] [SerializeField]
 		private GameObject larvae = null;
 
+		[Tooltip("The SpriteHandler the facehugger should use once it successfully impregnates a host")] [SerializeField]
```

Comment:
all items should only use one sprite handler except for layered sprites. Could you get the sprite handler on awake instead?